about.html

```diff
@@ -12,96 +12,105 @@
 </head>
 
 <body>
-    <div class="container about-page">
+    <div class="container-about-page">
+      <header>
+        <img src="C:\Users\djzar\OneDrive\Desktop\Lambda School_WebPT5_projects\Sprint-Challenge--DJZ\img\lambda-black.png">
+
       <nav class="nav-about">
         <div class="links-about">
-          <a class="nav-link-about" href="">Home</a>
+          <a class="nav-link-about" href="index.html">Home</a>
```

Comment:
Perfect reference to your index.html file. One thing I would like to mention as well is that the nav-link-about class is not needed if you are styling the navbar the same way in both files. You can use the same class name for both.